pkg/controller/application/application_controller.go

```diff
@@ -24,6 +25,7 @@ import (
 	shippererrors "github.com/bookingcom/shipper/pkg/errors"
 	apputil "github.com/bookingcom/shipper/pkg/util/application"
 	releaseutil "github.com/bookingcom/shipper/pkg/util/release"
+	rolloutBlockOverride "github.com/bookingcom/shipper/pkg/util/rolloutblock"
```

Comment:
I'd suggest keeping the name all-lower-case as it distinguishes them visually from variables.